path_planning:
  ros__parameters:
    #triangulation
    maximum_angle: 90
    maximum_edge_angle: 60
    maximum_distance: 5
    target_depth: 6

    #penalties
    same_edge_penalty: 1000
    length_penalty: 1.0
    angle_penalty: 1.0
    total_length_reward: 1.0

    #path planning
    selection_radius_small: 5
    selection_radius_big: 10
    selection_angle: 80

